packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitSpellTracker.tsx

```diff
@@ -62,7 +63,7 @@ function computePercentCDRemaining(
   currentTimeOffset: number,
   spellData: Record<string, IMinedSpell>,
 ): { cooldown: CooldownInfo; lastCastTimestampOffset?: number } {
-  const cooldown = spellData[spellId]?.cooldownSeconds || spellData[spellId]?.charges?.chargeCooldownSeconds || 30;
+  const cooldown = spellData[spellId]?.charges?.chargeCooldownSeconds || spellData[spellId]?.cooldownSeconds || 29.9;
```

Comment:
why 29.9?

Reply:
It makes it clear when debugging that the default value got used but visually will have no effect